src/components/markdown/Renderer.tsx

```diff
@@ -172,7 +172,7 @@ export default function Renderer({ content, disallowBigEmoji }: MarkdownProps) {
                                 try {
                                     const url = new URL(href, location.href);
 
-                                    if (url.hostname === location.hostname) {
+                                    if (url.hostname.replace(/^[^.]+\./g, "") === location.hostname.replace(/^[^.]+\./g, "") && ["nightly.", "app.", "local."].includes(url.hostname.match(/^[^.]+\./g)?.[0] ?? "")) {
```

Comment:
This feels a bit excessive, couldn't it just be done with a whitelist?

Reply:
Yeah it is, initially in the first commit I was just hoping for a quick regex strip. Then I seen there was the sun domains for the docs etc that failed. I'll update this PR with the changes soon.